cloudformation-templates/zs_cc_cf_template_zscc_macro.yaml

```diff
@@ -230,7 +230,7 @@ Resources:
                 filters = [{"Name": "product-code", "Values":[ImageCode]}]
                 response = sorted(client.describe_images(Owners=["aws-marketplace"], Filters=filters).get("Images"), key=lambda x:  x.get("CreationDate"))
                 
-                SecretManagerSecretName = event["templateParameterValues"].get("SecretManagerSecretName", "ZS/CC/Credentials")
+                SecretManagerSecretName = event["templateParameterValues"].get("SecretManagerSecretName", "ZS/CC/credentials")
```

Comment:
I got a comment for terraform too just the other day from the field where "ZS/CC/Credentials" was deemed confusing where they thought they needed "ZS/CC" as a prepend to the actual secret manager name.

What if we just put "Secret Name" in quotes?

Reply:
Not a bad idea, except I wonder if I should put in in the description of the parameter rather than default it to "SecretName" and having to verify that it is not "SecretName" in the code.

Reply:
do you have to default it to something? Or can you leave it as a required input?

Reply:
that deviates from existing behavior. If there is nothing specified, or if the field is cleared out, then we fall back to ZS/CC/credentials

Reply:
let's take it up later. I don't think changing this is impactful for the PR feature in question